teleirc.js

```diff
@@ -106,10 +106,15 @@ tgbot.on('message', function (msg) {
         if (msg.text === undefined) {
 
             // Check if this message is a new user joining the group chat:
-            if (msg.new_chat_member) {
+            if (msg.new_chat_member  && config.irc.showJoinMessage) {
                 let username = msg.new_chat_member.username;
                 let first_name = msg.new_chat_member.first_name;
                 ircbot.say(config.channel, "New user " + first_name + " ( @" + username + ") has joined the Telegram Group!");
```

Comment:
I feel like "New user" (and below, "user") prefacing the username of the join or leave is redundant.